UI/CameraCaptureUI/CameraCaptureUISample/CameraCaptureUISample.Wasm/CameraCaptureUISample - Backup.Wasm.csproj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.NET.Sdk.web">
```

Comment:
Remove this file.

Reply:
Strange. Maybe a VS crash colateral effect. Didn't see that.